Makefile

```diff
@@ -27,14 +27,14 @@ lint-prepare:
 
 .PHONY: lint
 lint: vendor
-	@gometalinter --cyclo-over=20 --deadline=2m $$(glide novendor)
+	@gometalinter --cyclo-over=20 --deadline=2m $$(dep ensure -v)
```

Comment:
I think the result of $(dep ensure -v) doesn't give equivalent result to the $(glide novendor)

I've seen using  `gometalinter --vendor ./...` should works